docs/examples/github/workflow-custom-refs-audit.yml

```
@@ -0,0 +1,74 @@
+name: Shiplog Audit (custom refs)
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)